packages/core/src/shared/plugins/suggestion/components/SuggestionGroup.tsx

```diff
@@ -70,23 +52,47 @@ function SuggestionComponent<T extends SuggestionItem>(
     }
   }, [isButtonSelected]);
 
+  const Icon = props.item.icon;
+
   return (
-    <div className={styles.buttonItem}>
-      <ButtonItem
-        isSelected={isButtonSelected} // This is needed to navigate with the keyboard
-        iconBefore={getIcon(props.item, isButtonSelected)}
-        onClick={(_e) => {
+    <MantineProvider
```

Comment:
As we use this in two places, should we maybe refactor to something like:

`<BlockNoteMantineProvider>...</BlockNoteMantineProvider>`?

Reply:
I think we still want to use a separate `MantineProvider` component for the Suggestions plugin since the Menu component used in the Suggestions has different styling to e.g. the Menu component used in the Toolbar. That being said, the `MantineProvider` component should probably wrap the whole Suggestions dropdown, not just `SuggestionGroup`.